src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -120,6 +133,12 @@ export class Lilies extends CoreGauge {
 
 	}
 
+	private checkAfflatusAvailability() {
+		if (this.lilyGauge.value !== 0 && this.bloodLilyGauge.value === 0) {
```

Comment:
We should be checking if the blood lily gauge is not capped rather than if its empty. We are fine to use a lily if we are at 2 blood lily for example.
The gauge is set up to cap at 3 and add 1 every normal lily usage.

Reply:
Yeah I'm just dumb and forgot that it takes three charges to create a blood lily 😅 it's been a minute